plugins/plugin-nodejs-debugger/che-plugin-nodejs-debugger-server/src/main/java/org/eclipse/che/plugin/nodejsdbg/server/NodeJsDebugProcess.java

```diff
@@ -107,115 +99,79 @@ void stop() {
             if (!executor.awaitTermination(10, TimeUnit.SECONDS)) {
                 executor.shutdownNow();
                 if (!executor.awaitTermination(10, TimeUnit.SECONDS)) {
-                    LOG.warn("Unable to terminate main pool");
+                    LOG.warn("Unable to terminate main pool.");
                 }
             }
         } catch (InterruptedException e) {
-            LOG.warn(e.getMessage());
+            if (!executor.isShutdown()) {
+                LOG.warn("Unable to terminate main pool.");
+            }
```

Comment:
Please handle `InterruptedException` and set interrupted flag. Why after interrupt you wait up to 10 seconds below?